ui-spacetimechart/src/components/PathLayer.tsx

```diff
@@ -237,18 +245,20 @@ export const PathLayer: FC<PathLayerProps> = ({
       ctx.save();
       ctx.translate(position.x, position.y);
       ctx.rotate(angle);
-      ctx.font = `${FONT_SIZE}px IBM Plex Mono`;
+      ctx.font = font;
       ctx.textAlign = 'start';
 
       const dx = 5;
       const dy = angle >= 0 ? -5 : 15;
       const padding = 2;
       const measure = ctx.measureText(label);
       const w = measure.width + 2 * padding;
-      const h = FONT_SIZE + 2 * padding;
-      ctx.fillStyle = WHITE_75;
+      const h = measure.actualBoundingBoxAscent + measure.actualBoundingBoxDescent + 2 * padding;
+      ctx.globalAlpha = 0.75;
+      ctx.fillStyle = background;
       ctx.fillRect(dx - padding, dy - h + padding, w, h);
       ctx.fillStyle = labelColor;
+      ctx.globalAlpha = 1;
       ctx.fillText(label, dx, dy - FONT_SIZE / 4);
```

Comment:
No, since the above is not a font size - but indeed, since a custom theme can override the actual font size, we should not use `FONT-SIZE`. I'll fix that :+1: